frontend/src/components/UserSearch.vue

```diff
@@ -171,6 +171,13 @@
           loading-text="Searching for users"
           v-on:click:row="selectUser"
         >
+          <!-- https://stackoverflow.com/questions/61394522/add-hyperlink-in-v-data-table-vuetify -->
+          <template #item.username="{ item }">
+            <a target="_blank" :href="`#/users/${item.id}`" v-on:click="openNewTab">
+              {{ item.username }}
+            </a>
+            <v-icon small>mdi-open-in-new</v-icon>
+          </template>
```

Comment:
We customize the `v-data-table` by using the "item slot". This renders the item as an `href` link instead of the default text. (I find this pretty crazy stuff. We get away from PrimeFaces and JSF for the sweet purity of HTML and Javascript, and find ourselves immediately mired in SPA's own hocus pocus.)

The `v-on:click="openNewTab"` event sets a flag to prevent the default data table's `v-on:click:row="selectUser"` from running, otherwise it opens a new tab _and_ loads the details in the same tab. I couldn't think of a better way to do this. I tried figuring out a way to determine what the user click inside the `selectUser` method, but couldn't find anything.